<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cout_person</class>
 <widget class="QWidget" name="cout_person">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>录入信息</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>生日：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>电话：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>电子邮箱：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="year">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="day">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="month">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="phone">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="email">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>你和他一起上的学校/共事的公司/认识的地点/称呼</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="scpc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="classmate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>同学</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="collaege">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>同事</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="the_friend">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>朋友</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="relative">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>亲戚</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>他/她和你的关系：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
